bin/lint.rb

```diff
@@ -684,21 +684,6 @@ def self.check_bolded_heading(contents)
     end
   end
 
-  def self.zenodo_api(contents)
-    find_matching_texts(contents, /(zenodo\.org\/api\/files\/)/)
-      .map do |idx, _text, selected|
-      ReviewDogEmitter.error(
-        path: @path,
-        idx: idx,
-        match_start: selected.begin(1),
-        match_end: selected.end(1) + 1,
-        replacement: nil,
-        message: 'The Zenodo.org/api URLs are not stable, you must use a URL of the format zenodo.org/record/..., apologies we cannot fix automatically.',
-        code: 'GTN:032'
-      )
-    end
-  end
-
   def self.fix_md(contents)
```

Comment:
this PR seems to have the same issue sadly, but if you can fix it i'll get it reviewed asap :)